### DISCUSSION TITLE
Unofficial layers

### DISCUSSION THREAD

**Discussion by dalanicolai** [syl20bnr/spacemacs#14757] [Show and tell]
Although we can rely on google to find unofficial layers, I think it would be nice to have a page somewhere where users can post their layers they would like to share, but did not yet put all effort to make it into an official layer.

Maybe we can just post links to layers in this thread (users could even up/downvote), and maybe link to this thread from the [Spacemacs layers page](https://develop.spacemacs.org/layers/LAYERS.html) (so I will categorize this post under "Show and tell"). Or should I just create a PR (see https://github.com/rajasegar/awesome-spacemacs/issues/3) at [awesome-spacemacs](https://github.com/rajasegar/awesome-spacemacs) and we create a link to there? (actually I found awesome-spacemacs only just now, while checking if I am not reinventing the wheel here. Maybe this last suggestion is my preferred solution, despite that repo is not maintained by the "official Spacemacs team")

Any opinions? Maybe this is no good idea at all, and this discussion should be removed? Anyway, all opinions are welcome...

**Comment by dalanicolai** [comment]
https://github.com/dalanicolai/pdf-layer (includes continuous-scroll and epub support for pdf-tools, see [image-roll](https://github.com/dalanicolai/image-roll.el)).
https://github.com/dalanicolai/djvu-layer (merged into Spacemacs https://github.com/syl20bnr/spacemacs/pull/14920)
https://github.com/dalanicolai/robot-framework-layer
https://github.com/dalanicolai/maxima-layer
https://github.com/dalanicolai/ebib-layer (merged into Spacemacs https://github.com/syl20bnr/spacemacs/pull/14876)
https://github.com/dalanicolai/calibre-layer
https://github.com/dalanicolai/toc-layer (implements toc-mode. toc-mode has been moved to elpa under the name [doc-toc](https://elpa.gnu.org/packages/doc-toc.html). However, the toc-mode package is also still available, so that probably for most Spacemacs users it is easiest, and perfectly fine, to just use this outdated layer)
https://github.com/dalanicolai/info-layer (evilifies info-mode and adds a transient state using hydra)
https://github.com/dalanicolai/gitter-layer
https://github.com/dalanicolai/mathematica-layer

**Comment by lebensterben** [comment]
@dalanicolai 
I think you can make a new wiki page
https://github.com/syl20bnr/spacemacs/wiki

**Reply by dalanicolai** [reply]
Sounds like a good idea... I will probably do that at some point when I find the time. Anyway, they are not very responsive at [awesome-spacemacs](https://github.com/rajasegar/awesome-spacemacs/pull/7).

**Reply by lebensterben** [reply]
doesn matter if we have our own listing
